src/com/lftechnology/java/training/niraj/ems/utils/DbFacade.java

```diff
@@ -0,0 +1,372 @@
+     * 
+     * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+     * @param connection
+     * @param table
+     * @param attributes
+     * @param conditions
+     * @return
+     * @throws SQLException
+     */
+    public static PreparedStatement createSelectStatement(Connection connection, String table, String[] attributes,
+            Map<String, String> conditions, Operators operator) throws SQLException {
+
+        StringBuilder query = new StringBuilder();
+        query.append(DbFacade.addSelectClause(attributes, table));
+        query.append(DbFacade.addWhereClause(conditions, operator));
+        PreparedStatement stmt = connection.prepareStatement(query.toString());
+        return setStatementParameters(stmt, conditions);
+
+    }
+
+    /**
+     * Create select statement
+     * 
+     * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+     * @param connection
+     * @param table
+     * @param attributes
+     * @param conditions
+     * @return
+     * @throws SQLException
+     */
+    public static PreparedStatement createSelectStatement(Connection connection, String table, String[] attributes,
+            Map<String, String> conditions) throws SQLException {
+
+        StringBuilder query = new StringBuilder();
+        query.append(DbFacade.addSelectClause(attributes, table));
+        query.append(DbFacade.addWhereClause(conditions, Operators.AND));
+        PreparedStatement stmt = connection.prepareStatement(query.toString());
+        return setStatementParameters(stmt, conditions);
+
+    }
+
+    /**
+     * Creates count statement with condition
+     * 
+     * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+     * @param connection
+     * @param table
+     * @param conditions
+     * @return
+     * @throws SQLException
+     */
+    public static PreparedStatement createCountStatement(Connection connection, String table, Map<String, String> conditions,
+            Operators operator) throws SQLException {
+        StringBuilder query = new StringBuilder();
+        query.append(Constants.SELECT_COUNT_QUERY);
+        query.append(table);
+        query.append(DbFacade.addWhereClause(conditions, operator));
+
+        PreparedStatement stmt = connection.prepareStatement(query.toString());
+        return setStatementParameters(stmt, conditions);
+    }
+
+    /**
+     * Creates count statement with condition
+     * 
+     * @author Niraj Rajbhandari <nirajrajbhandari@lftechnology.com>
+     * @param connection
+     * @param table
+     * @param conditions
+     * @return
+     * @throws SQLException
+     */
+    public static PreparedStatement createCountStatement(Connection connection, String table) throws SQLException {
+        StringBuilder query = new StringBuilder();
+        query.append(Constants.SELECT_COUNT_QUERY);
+        query.append(table);
+
+        PreparedStatement stmt = connection.prepareStatement(query.toString());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "stmt". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1488)